translation/v3_to_v2.xsl

```diff
@@ -49,11 +49,7 @@ Version: 0.1.0
   <xsl:template match="auc:MeasureName[. = 'Insulate attic hatch']">
     <xsl:copy>Insulate attic hatch / stair box</xsl:copy>
   </xsl:template>
-  <!-- 3. Map enumeration "MROE" to "MORE" under auc:eGRIDRegionCode -->
-  <xsl:template match="auc:eGRIDRegionCode/auc:MeasureName[. = 'MROE']">
-    <xsl:copy>MORE</xsl:copy>
-  </xsl:template>
```

Comment:
What's the reason for removing this?

Reply:
This is a typo option and we have added the correct option (`MROE`) in 2.4, so we don't need this now.